test_priorityq.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Pytest module for 'priorityq.py'
+"""
+
+from __future__ import unicode_literals
+import pytest
+from priorityq import PriorityQ, Item
+
+
+@pytest.fixture(scope="module")
+def build_priorityq():
+    built = PriorityQ()
+    jim = Item("jim", 5)
+    allen = Item("allen", 1)
+    number2 = Item(2, 2)
+    number3 = Item(3, 3)
+
+    built.insert(jim)       # heapList = [0, 5]
+    built.insert(allen)     # heapList = [0, 1, 5]
+    built.insert(number2)   # heapList = [0, 1, 5, 2]
+    built.insert(number3)   # heapList = [0, 1, 3, 2, 5]
```

Comment:
I like these comments. It shows your reader that you know exactly what each step should be doing.